lib/ms.js

```diff
@@ -23,6 +23,10 @@ function docpress (cwd, options) {
     .metadata(meta)
     .ignore('node_modules')
     .ignore(`!**/{${meta.docs}{,/**/*},*.md}`)
+    
+  if (typeof meta.ignore == 'object' && meta.ignore.forEach) {
```

Comment:
I think we can use `Array.isArray(meta.ignore)` and then use `forEach`.

Reply:
Good point. Didn't realize `Array.isArray` existed. Learn something new everyday!